src/Apps/MyCollection/Routes/EditArtwork/MyCollectionEditArtwork.tsx

```diff
@@ -136,7 +136,7 @@ export const MyCollectionEditArtworkFragmentContainer = createFragmentContainer(
   {
     artwork: graphql`
       fragment MyCollectionEditArtwork_artwork on Artwork {
-        artist {
+        artist(shallow: true) {
```

Comment:
Wow, this one really demonstrates how _much_ data as accessible from the embedded artist representation. 

**question (non-blocking):** I double-checked this one in my API client to be sure, but I'm curious what the strategy was for verifying these changes?

Reply:
It's a multi-step, multi-language process. 😓

First I had to check [how each Metaphysics property is resolved](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/artist/index.ts). Often those are based directly on a corresponding API property, but sometimes they perform additional data-loading or are synthesized based on a few upstream properties. In any case, I'd then have to verify that _all_ of the necessary input properties are available within [artists' short JSON from Gravity](https://github.com/artsy/gravity/blob/a5dd184838ab0ed6a6b4f215ce7dedda6546c50e/app/models/domain/artist.rb#L382). When in doubt I visually confirmed a few locations still rendered the expected data locally, but I didn't check every case that way.